Lib/test/test_importlib/test_util.py

```diff
@@ -757,5 +757,49 @@ def test_source_from_cache_path_like_arg(self):
  ) = util.test_both(PEP3147Tests, util=importlib_util)
 
 
+class MagicNumberTests:
```

Comment:
Yeah, this module does some interesting things to create "real" test classes in different configurations, but we don't need those kinds of tricks for the new test.